LayoutTestBase/Config/LYTConfig.h

```diff
@@ -107,6 +116,21 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic) CGFloat cgFloatEpsilon;
 
+
+/**
+ Default snapshots to save per method.
+ */
+#define SaveUnlimitedSnapshotsPerMethod -1
```

Comment:
#define doesn't work in swift projects. I think we need to make this an extern NSInteger like the constants in LYTDeviceConstants.h

Also, since it's in the global space, we should prefix with LYT